content/docs/1.7.0/snapshots-and-backups/csi-volume-clone.md

```diff
@@ -5,6 +5,8 @@ weight: 3
 ---
 
 Longhorn supports [CSI volume cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/).
+
```

Comment:
How about changing the layout to

```
## Volume Cloning
### Clone a Volume Using YAML
### Clone a Volume Using the Longhorn UI
### Clone Volumes ...
...

## Volume Creation
```

Reply:
Want to put UI actions under the same block
```
## Volume Cloning
### Clone a Volume Using YAML
....
### Clone Volume Using the Longhorn UI
#### Clone a volume
#### Clone a Volume Using a Snapshot
#### Clone Multiple Volumes (Bulk Cloning)
...
## Volume Creation
```